SPECIFICATION.md

```diff
@@ -1,179 +1,175 @@
 
-A Ceramic document consists of an append-only log of records where updates are signed by a decentralized identifier (DID), validated against a set of state transition rules, then anchored on a blockchain. *Doctypes* specify the explicit rules that govern what constitutes a valid update (state transition) to the document. These could be direct events, such as signatures from the owners, or indirect events, such as the change in state of another document. Doctypes allow Ceramic nodes to manage and verify the state of a given document in a decentralized way without centralized servers or middlemen. When a valid update is made to a particular document, the node then gossips the update to the rest of the network and any nodes interested in this document will update the state of the given document on their node. As a result, any node can receive a query for a document and return the correct state even if it does not maintain the state of the document itself. The protocol is agnostic as to which blockchain is used for anchoring, and developers could choose to have their document anchored on the blockchain of their choice.
+A Ceramic stream consists of an append-only log of commits where updates are signed by a decentralized identifier (DID), validated against a set of state transition rules, then anchored on a blockchain. *Streamtypes* specify the explicit rules that govern what constitutes a valid update (state transition) to the stream. This architexture allows Ceramic nodes to manage and verify the state of a given stream in a decentralized way without centralized servers or middlemen. When a valid update is made to a particular stream, the node then gossips the update to the rest of the network and any nodes interested in this stream will update the state of the given stream on their node. As a result, any node can receive a query for a stream and return the correct state even if it does not maintain the state of the stream itself. As long as at least one node on the network continues to maintain state for a given stream, that stream remains available to *any* node on the network.
```

Comment:
Minor typo: `architexture` -> `architecture`

(Although I like the word `architexture` 😁)